scalableminds/webknossos
# Add Markdown to taskType descriptions

### Mailable description of changes:
 - TaskType descriptions can now be written in Markdown

### Steps to test:
- Edit a taskType's description, e.g.,
```
Please trace the indicated axon in both directions. 

If you encounter a presynaptic zone within the axon (look for the vesicles):
1) Create a branch-point
2) Set another node towards the contact area of the two processes and label it according to the following scheme:
          syn spine: Synapse onto a spine 
          syn shaft Synapse onto a shaft
          syn soma Synapse onto a soma
3) Trace the active area in a view-port orthogonal to the contact area. The active area is defined as the area where you can see vesicles in the presynaptic compartment as well as a post-synaptic density. Do this for all slices where you can still see the active zone of the synapse.
5) Once you are finished annotating: Jump back to the branch-point & continue tracing. (Take careful not to choose the WRONG direction!)

Contact email if you encounter any problems or have questions. Also if you have doubts about doing the right thing, save your tracing and drop me a line. Than i can have a look usually within hours. If explanations are not enough we can make an appointment at the MPIN to give you an introduction.

Thanks a lot for tracing!
```
- Click `Get Next Task` in Dashboard and behold the pretty task description

### Issues:
- fixes #2027

------
